tests/test_batch.py

```diff
@@ -126,68 +130,6 @@ def test_cancel_job_sdk(self, request_mock):
             == f"{self.sdk._client.endpoints.core}/api/v1/jobs/{self.job_id}/cancel"
         )
 
-    @pytest.mark.skip(reason="Not enabled during Iroise MVP")
```

Comment:
I'm wondering about the skipped Iroise MVP tests, they have been here for months, we can remove them right ?

Reply:
shouldn't it be the opposite? Now after Iroise MVP the functionality tested by these tests should be enabled?

They might also just be so old now that it's not needed of course

Reply:
Actually I was thinking about it 5 mins ago because there is a problem, some methods aren't tested at all like declare_complete or add_jobs, so I see 1 option: update the tests and make them work back cause they aren't working if we remove the skip decorator

Reply:
This feature has not been properly reimplemented. Open batches are not properly supported by the dgx-connector nor the new version of pasqman. While this is not the case I don't think you should bother re-enabling these tests; you may however update the skip message as it is out of date.

AFAIK the first applications we wanted to run on QPU & emulators didn't require open batches which is why it was de-prioritized. The new workload API may support it in the near future.

Reply:
The thing is why do we still have these features if they aren't tested ? shouldn't they be non-usable in that case ?

Reply:
By updating the add_jobs methods I discovered that there is a problem in my code for example, so should I update the tests or should we mark these methods as non-usable ?

Reply:
To be fair the unit tests could be re-enabled as the API response is mocked. So if it helps us catching bugs, you can give it a try

In practice the API will reject the request as batches are automatically closed for now but the message should be clear for the user